pyproject.toml

```
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["setuptools", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pysatCDAAC"
+version = "0.0.3.0"
```

```suggestion
version = "0.0.3"
```
The extra digit was to create a unique identifier for the test pypi